README.md

```diff
@@ -80,16 +80,18 @@ Adapt the instructions to suit your cmake generator and operating system.
 Build and install the KDDockWidgets framework:
 
 ```
-$ cmake -G Ninja -DCMAKE_INSTALL_PREFIX=/path/where/to/install ../path/to/kddockwidgets
```

Comment:
this was fine, as it worked with both in-source and out-of-source builds
the new line is less flexible and works only with out-of-source